README.md

```diff
@@ -1,138 +1,187 @@
+confidence.
 
-sgrep, for syntactical \(and occasionnally semantic\) grep, is a tool to help find bugs by specifying code patterns using a familiar syntax. The idea is to mix the convenience of grep with the correctness and precision of a compiler frontend.
+**Try it now:** [https://sgrep.live](https://sgrep.live/)
 
-## Quick Examples
+## Overview
 
-| **pattern** | **will match code like** |
-| :--- | :--- |
-| `$X == $X` | `if (node.id == node.id): ...` |
-| `foo(kwd1=1, kwd2=2, ...)` | `foo(kwd2=2, kwd1=1, kwd3=3)` |
-| `subprocess.Popen(...)` | `import subprocess as s; s.Popen(['foo'])` |
-
-→ [see more examples in the sgrep-rules registry](https://github.com/returntocorp/sgrep-rules)
+Language support:
 
-## Supported Languages
-
-| **javascript** | **python** | **go** | **java** | **c** | **ruby** | **scala** |
+| **Python** | **Javascript** | **Go &nbsp; &nbsp; &nbsp;** | **Java &nbsp;** | **C &nbsp; &nbsp; &nbsp; &nbsp;** | **Typescript** | **PHP &nbsp; &nbsp;** |
 | :--- | :--- | :--- | :--- | :--- | :--- | :--- |
-| ✅ | ✅ | ✅ | ✅ | ✅ | coming | coming |
-
-→ see full language support details in [matrix.md](docs/matrix.md)
+| ✅ | ✅ | ✅ | ✅ | ✅ | Coming... | Coming... |
 
-## Meetups
+Simple patterns for finding code in many languages:
 
-Want to learn more about sgrep? Check out these [slides from the r2c February meetup](https://r2c.dev/sgrep-public2.pdf)
+| **Pattern** | **Matches** |
+| :--- | :--- |
+| `$X == $X` | `if (node.id == node.id): ...` |
+| `requests.get(..., verify=False, ...)` | `requests.get(url, timeout=3, verify=False)` |
+| `os.system(...)` | `from os import system; system('echo sgrep')` |
+| `$ELEMENT.innerHTML` | ``el.innerHTML = "<img src='x' onerror='alert(`XSS`)'>";`` |
+| `$TOKEN.SignedString([]byte("..."))` | `ss, err := token.SignedString([]byte("HARDCODED KEY"))` |
 
 ## Installation
 
-Too lazy to install? Try out [sgrep.live](https://sgrep.live)
-
-### Docker
-
-`sgrep` is packaged within a [docker container](https://hub.docker.com/r/returntocorp/sgrep), making installation as easy as [installing docker](https://docs.docker.com/install/).
+For the best experience, simply install `sgrep` with [Docker](https://docs.docker.com/install/):
 
-## Quickstart
-
-```bash
-docker pull returntocorp/sgrep
+```
+$ docker pull returntocorp/sgrep
+```
 
-cd /path/to/repo
-# generate a template config file
-docker run --rm -v "${PWD}:/home/repo" returntocorp/sgrep --generate-config
+`sgrep` binaries can also be installed on Ubuntu:
 
-# look for findings
-docker run --rm -v "${PWD}:/home/repo" returntocorp/sgrep
+```
+$ wget https://github.com/returntocorp/sgrep/releases/download/v0.4.9/sgrep-0.4.9-ubuntu-16.04.tgz
+$ wget https://github.com/returntocorp/sgrep/releases/download/v0.4.9/sgrep-0.4.9-ubuntu-16.04.tgz.sha256
+$ sha256sum --check sgrep-0.4.9-ubuntu-16.04.tgz.sha256
+$ tar xf sgrep-0.4.9-ubuntu-16.04.tgz
+$ ./sgrep-lint-files/sgrep-lint --help
```

Comment:
IMO we should not be recommending this path (in the README) until it is as easy as `apt-get install semgrep` @DrewDennison  thoughts?

Reply:
I'll remove for now, we can re-add when we have a clearer picture.